test/common_methods.py

```diff
@@ -753,7 +753,7 @@ def test_many_files(self):
 
         test_dir = os.path.join(self.import_dir, 'many')
         files = []
-        for x in range(0, 100):
+        for x in range(0, 15):
```

Comment:
This isn't needed now a the bug has been fixed in CaRT.  I'll update the CaRT ref and then we can just process the log changes in this PR.